<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="replayPushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leaderboardsPushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Leaderboards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="playerInformationGroupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Players</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cardsGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>451</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
